mautrix_signal/matrix.py

```diff
@@ -54,6 +54,16 @@ def __init__(self, bridge: "SignalBridge") -> None:
 
         super().__init__(bridge=bridge)
 
+    async def handle_invite(
+        self, room_id: RoomID, user_id: UserID, inviter: u.User, event_id: EventID
+    ) -> None:
+        user = await u.User.get_by_mxid(user_id, create=False)
+        if not user:
+            return
+        portal = await po.Portal.get_by_mxid(room_id)
+        if portal:
+            await portal.handle_matrix_invite(inviter, user)
```

Comment:
This should probably catch `RejectMatrixInvite` and send an error message (or at least log and ignore rather than letting it propagate). Something like this:

```suggestion
        if portal and not portal.is_direct:
            try:
                await portal.handle_matrix_invite(inviter, user)
            except RejectMatrixInvite as e:
                await portal.main_intent.send_notice(portal.mxid, "Failed to invite {user.mxid} on Signal: {e}")
```

(needs an import for `RejectMatrixInvite`)